components/director/Dockerfile

```diff
@@ -38,7 +38,7 @@ RUN mkdir /app && mv ./director /app/director \
   && mv ./systemfetcher /app/systemfetcher \
   && mv ./licenses /app/licenses
 
-FROM alpine:3.15.0
+FROM alpine:3.15.3
```

Comment:
golang

Reply:
It is not trivial and require refactoring. We agree with @NickyMateev to do this in separate task.